core/src/test/java/org/elasticsearch/index/query/FieldMaskingSpanQueryBuilderTest.java

```diff
@@ -83,7 +83,7 @@ public void testValidate() {
         if (randomBoolean()) {
             fieldName = "fieldName";
         } else {
-            fieldName = randomBoolean() ? "" : null;
+            fieldName = "";
```

Comment:
We want to test validate() on a query that we can successfully construct. We have null-checks for fieldname here now.